docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -97,6 +97,43 @@ supports the following features:
   :ref:`iceberg-materialized-views`
 * :ref:`sql-views-management`
 
+.. _iceberg-alter-table-execute:
+
+ALTER TABLE EXECUTE
+^^^^^^^^^^^^^^^^^^^
+
+The connector supports the following commands for use with
```

Comment:
I would simplify .. something like this:

The connector supports collapsing files in a table into fewer larger files with identical data. This operation improves performance and reduces disk usage.

All files with a size below the optional ``file_size_threshold`` parameter are collapsed. The default value for the threshold is ``100MB``:

.. code-block:: sql

    ALTER TABLE test_table EXECUTE OPTIMIZE

Reply:
I did reword the paragraph with your suggestion, but kept the reference towards partitioning. I think it is important to mention that the collapsing takes place per partition in case of partitioned tables.